homeassistant/components/refoss/coordinator.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, hass: HomeAssistant, device: BaseDevice) -> None:
             hass,
             _LOGGER,
             name=f"{DOMAIN}-{device.device_info.dev_name}",
-            update_interval=timedelta(seconds=15),
+            update_interval=timedelta(seconds=5),
```

Comment:
This seems an unrelated change to what is being described in this PR?

Reply:
User feedback:   hope this time interval is shorter ： https://github.com/ashionky/refoss_ha/issues/6

Reply:
HI, @frenck 
Is there any problem with this PR?
Looking forward to your guidance.

Reply:
Let's do this in a separate PR. I have a hard time believing that this is not possible without service call. My suspicion would be that the machine they are running this on is doing too much at the same time and can't keep up with the 5 seconds. Because 5 seconds is very very low for HA standards

Reply:
> Let's do this in a separate PR. I have a hard time believing that this is not possible without service call. My suspicion would be that the machine they are running this on is doing too much at the same time and can't keep up with the 5 seconds. Because 5 seconds is very very low for HA standards

Ok, I'll deal with this feature later.
Our device supports 5-second requests (the app also requests devices at 5-second intervals)
Users have previously reported this situation.

Reply:
> Let's do this in a separate PR. I have a hard time believing that this is not possible without service call. My suspicion would be that the machine they are running this on is doing too much at the same time and can't keep up with the 5 seconds. Because 5 seconds is very very low for HA standards

Can I submit a separate PR for this modification point？

Reply:
You're always free to open a PR, but I think that 5 seconds is very very low and not useful for the majority of users